scripts/lib/api/Pkg.ts

```diff
@@ -27,6 +27,16 @@ export class ReleaseNotesConfig {
   }
 }
 
+export class TocConfig {
```

Comment:
What do you think about moving this to `TocGrouping.ts` and making it a `type` instead of a `class`? Something like:
```ts
export type TocConfig = {
  truncate?: boolean;
  tocGrouping?: TocGrouping;
};
```

Reply:
The class is nice because it allows us to use default args for the constructor. I used this same approach for `ReleaseNotesConfig`: https://github.com/Qiskit/documentation/pull/1305/commits/27b7a73fd21552ab17623d5bf23474a06926c05a

> What do you think about moving this to TocGrouping.ts

I could and we rename it to `TocConfig.ts`, but IMO the current spot is fine. `TocGrouping` is already pretty complex and focused entirely on the complex custom grouping mechanism. `Pkg.ts` is more generally about the config for a Pkg. Because `TocConfig` is such a simple class, I think it's fine to be near `Pkg`.

Reply:
Sounds good, thanks for the explanation!